eng/Version.Details.xml

```diff
@@ -123,7 +123,7 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>b47a997282b4d347cb079d26515ea4147e2ac548</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.Build.SharedFramework.Sdk" Version="5.0.0-beta.19515.1">
+    <Dependency Name="Microsoft.DotNet.Build.SharedFramework.Sdk" Version="5.0.0-beta.19517.8">
```

Comment:
Oh, whoops. Do you mind including this fix?

```suggestion
    <Dependency Name="Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk" Version="5.0.0-beta.19517.8">
```

Reply:
That'll do it haha. I'll add this in.